library/lms.c

```diff
@@ -499,13 +498,21 @@ static int get_merkle_path( mbedtls_lms_private_t *ctx,
                             unsigned int leaf_node_id,
                             unsigned char *path )
 {
-    unsigned char tree[MERKLE_TREE_NODE_AM_MAX][MBEDTLS_LMS_M_NODE_BYTES_MAX];
+    const size_t node_bytes = MBEDTLS_LMS_M_NODE_BYTES(ctx->params.type);
```

Comment:
```suggestion
    const size_t node_bytes = MBEDTLS_LMS_M_NODE_BYTES( ctx->params.type );
```

Reply:
We're [switching to the new style](https://github.com/Mbed-TLS/mbedtls/issues/6359) at the end of the month (or maybe first week of January). So let's not be picky about it now.